acceptance-tests/src/test-support/java/tech/pegasys/artemis/test/acceptance/dsl/ArtemisDepositSender.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2019 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.artemis.test.acceptance.dsl;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.testcontainers.containers.Network;
+import tech.pegasys.artemis.util.Waiter;
+
+public class ArtemisDepositSender extends Node {
+  private static final Logger LOG = LogManager.getLogger();
+
+  public ArtemisDepositSender(final Network network) {
+    super(network, ArtemisNode.ARTEMIS_DOCKER_IMAGE, LOG);
+  }
+
+  public void sendValidatorDeposits(final BesuNode eth1Node, final int numberOfValidators) {
```

Comment:
Should we require an `amount` to be set?

Reply:
At the moment its just sending the 32 ETH required to register a new validator. For bulk validator registration like this I think that makes sense - if you were depositing smaller amounts over time you'd likely be doing it for one specific validator.